### PR TITLE
change node_.get() to this in src/runtime/module.cc

### PR DESCRIPTION
**Pull request by zxy844288792**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by yongwww** [comment]
@zhiics @wweic @kevinthesun  pls take a look.

File: src/runtime/module.cc [hunk 1]
```diff
@@ -97,7 +97,7 @@ Module Module::LoadFromFile(const std::string& file_name,
 }
 
 bool Module::IsEmpty() const {
-  return node_.get() == nullptr;
+  return this->operator->() == nullptr;
```
Comment (yongwww) [inline src/runtime/module.cc:100]: does apache/tvm have this change?

Comment (zhiics) [reply src/runtime/module.cc:100]: I don't think Apache TVM has this. Can you just use `return this->defined();` as `Module` is an `ObjectRef`?

Comment (zxy844288792) [reply src/runtime/module.cc:100]: > does apache/tvm have this change?

No, apache/tvm does not have this. This is used for our TRT implementation.


> I don't think Apache TVM has this. Can you just use `return this->defined();` as `Module` is an `ObjectRef`?

Yes, I will try that.

**Review by reminisce — APPROVED** [review]

**Review by yongwww — APPROVED** [review]

**Pull request merged**